gcp/website/frontend_handlers.py

```diff
@@ -1044,11 +1081,11 @@ def _get_downstreams_of_bug(bug_id: str, bugs: dict[str,
   return downstreams
 
 
-def compute_downstream_hierarchy(target_bug_id: str,
-                                 downstreams: dict[str, list[str]]) -> str:
+def compute_downstream_hierarchy(
+    target_bug_id: str,
+    downstreams: dict[str, list[str]]) -> tuple[set[str], dict[str, set[str]]]:
```

Comment:
We can probably alias this big tuple into its own type so we can reuse it above in add_known_osv_bugs